.github/workflows/lint_master.yml

```diff
@@ -1,4 +1,4 @@
-name: Axivion
```

Comment:
What will happen to the Axivion job?

Reply:
It is here just a renaming with a more generic name